src/app/libp2p_helper/src/codanet.go

```diff
@@ -216,7 +248,13 @@ func (gs *CodaGatingState) InterceptAddrDial(id peer.ID, addr ma.Multiaddr) (all
 	_, exists := os.LookupEnv("CONNECT_PRIVATE_IPS")
 
 	// if we want to allow connecting to private IPs, and this addr is a private IP, allow
-	if exists && gs.AddrFilters.AddrBlocked(addr) {
+	isPriv, err := isPrivate(addr)
+	if err != nil {
+		gs.logger.Debugf("multiaddress does not contain IP: %v", addr)
```

Comment:
I think some of the multiaddrs folks use are dns ones. I think it is still safe to return false here — but just double checking that we’d want to log debug in this situation. Seems okay though?

Reply:
we're using DNS peers for the seed nodes and that connection specifically is failing right now, definitely worth making sure that DNS peers work as normal

Reply:
let me know if i should change this!